### ISSUE TITLE
 Waydroid remnants left behind in Desktop Mode

### ISSUE THREAD

**Issue by Fofer** [ryanrudolfoba/SteamOS-Waydroid-Installer#208]
Hi, this it's a very impressive project, thanks for all of your work on it!  Earlier this year, I installed Waydroid on a Steamdeck to check it out. I subsequently uninstalled it too though; I wasn't using it enough to justify the storage space.  Ever since then, unfortunately, I've seen these Waydroid apps in the "Lost & Found" Application menu and I can't figure out how to get rid of them.  The "Uninstall or Manage Add-ons" menu item doesn't do anything.  Attempting to open them gives me the same error, Plasma Desktop Workspace says it "could not find the program."

http://i.imgur.com/FIXCjpm.jpeg

Any assistance is appreciated, thanks!

**Comment by ryanrudolfoba** [comment]
you might be using an older version of my script that leaves the Waydroid / Android shortcuts behind. This has been fixed eversince the start of this year. To manually cleanup and remove those items -

`sudo rm -rf ~/.local/share/waydroid ~/.local/share/applications/waydroid*`

Be careful with that command make sure no typos especially the asterisk part there should be no space in it!

**Comment by Fofer** [comment]
Yes, I was one of the earlier installers of Waydroid for SteamDeck, IIRC it was shortly after the second release. Longer ago than I remember!  After I uninstalled it, I didn't pay much attention to future releases.  
Thanks for the help! I copy-pasted that command above into Konsole and that successfully emptied out every "Lost & Found" item that had been listed there, except for Waydroid itself.  Waydroid was the only one left behind.  It was still there after a reboot.  So I navigated to home/.local/share/applications/ and manually trashed it myself.  Now it too is gone from the Lost & Found.  Thanks again!
